### PR TITLE
Feat/game percentages eb

### PR DESCRIPTION
**Pull request by EricBelongea**
This PR will move methods created into the Game class. Refactor tests in game_spec.

### CODE REVIEW

File: spec/game_spec.rb
Comment (zkevkev) [inline spec/game_spec.rb:1]: Thank you for catching this error! We will have to make the same changes to both league_spec and season_spec.

File: lib/stat_tracker.rb
Comment (EricBelongea) [inline lib/stat_tracker.rb:1]: This was all moved into our Game class

File: lib/game.rb
Comment (zkevkev) [inline lib/game.rb:1]: This looks great! We might be able to use a helper method for #percentage_home_wins. We might also be able to look at using a module for some aspects of these methods.

**Review by zkevkev — APPROVED** [review]
Overall these look great. I think we could use a helper method to DRY this up a bit but fantastic work!

File: lib/stat_tracker.rb
Comment (billyburch) [inline lib/stat_tracker.rb:1]: Moving these into the Game class from StatTracker helps with overall readability and makes sense for big picture organization.

File: lib/game.rb [hunk 1]
```diff
@@ -1,7 +1,91 @@
...
+    @game_data.each do |game|
+      game.each do |key, value|
+        
+      end
+    end
+  end
+
+  # GAME STATS: Percentage Category; Home wins, Away wins, Ties. EB
+  def percentage_home_wins
+    home_wins = 0
```
Comment (vsanantone) [inline lib/game.rb:34]: I wonder if we could refactor this if time permits. I do like your perspective with the counters to keep track of movement. Let's see how this method unfolds.

File: lib/game.rb
Comment (EricBelongea) [reply lib/game.rb:1]: This is something that we should look into!!!

**Review by billyburch — APPROVED** [review]
I like the direction in which our overall organization is moving - the classes help to clarify how the project is laid out.

**Pull request merged**